assets/templates/dining_lists/dining_day.html

```diff
@@ -1,37 +1,65 @@
-
 {% extends 'base.html' %}
 {% load staticfiles %}
 {% load humanize %}
 
-{% block title %}Scala Dining {{ date }} {% endblock %}
+{% block title %}Scala Dining {{ date }}{% endblock %}
 
 {% block head %}
     <link rel="stylesheet" href="{% static 'css/dining_v2.css' %}">
 {% endblock %}
 
 {% block content %}
-    <div class="col-12 row p-0 m-0">
-
-        {% if previous_date %}
-            <a href="{% url 'day_view' day=previous_date.day month=previous_date.month year=previous_date.year %}"
-               class="col-2 btn btn-outline-primary">Back</a>
-        {% else %}
-            <a class="col-2"></a>
-        {% endif %}
-
-        <div class="text-center col-8 py-1 h5
-            {% if is_today %} font-bold {% endif %}">
-            <span class="align-middle">{{ date|date|capfirst }}</span>
+    <div class="row">
+        {# Button toolbar #}
+        <div class="col-lg-4">
+            <div class="row">
+                <div class="col-5">
+                    {# (upcoming is a class method, i.e. not specific to the instance) #}
+                    {% with u=date.upcoming %}
+                        <a href="{% url 'day_view' day=u.day month=u.month year=u.year %}"
+                           class="btn btn-block btn-outline-primary">
+                            {% if u == date.today %}Today{% else %}Upcoming{% endif %}
+                        </a>
+                    {% endwith %}
+                </div>
+                <div class="col-7">
+                    <div class="btn-group btn-block">
+                        {% with d=date.previous %}
+                            <a href="{% url 'day_view' day=d.day month=d.month year=d.year %}"
+                               class="btn btn-outline-primary">
+                                <i class="fas fa-chevron-left"></i>
+                                <span class="sr-only">Previous</span>
+                            </a>
```

Comment:
Currently dininglist transactions are pending for two days after which entries  should no longer be removed or added (because it won't represent the cost). To prevent abuse of the helping statistics a similar limit might also be needed on the stats (to prevent dining list owners tweaking their own entries on very old dininglists. Do we want to represent the unchangable limits to the UI (possibly limit going backwards for more than two days)?

Reply:
Don't see any need for indicating it in the UI, if you want we can show a similar warning. If you want to hide the previous button we should instead block access altogether to these dates. We could do that by making the date sequence bounded